### PR TITLE
new package - mingw-w64-python-idna_ssl

### PR DESCRIPTION
**Pull request by JPeterMugaas**

### CODE REVIEW

File: mingw-w64-python-idna_ssl/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Maintainer: Some One <some.one@some.email.com>
+
+_realname=idna_ssl
+_intname=idna-ssl
+pkgbase=mingw-w64-python-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-python-${_realname}")
+pkgver=1.1.0
+pkgrel=1
+pkgdesc="Patch ssl.match_hostname for Unicode(idna_ssl) domains support (mingw-w64)"
+arch=('any')
```
Comment (Biswa96) [inline mingw-w64-python-idna_ssl/PKGBUILD:10]: Need mingw_arch array.

**Comment by lazka** [comment]
Why is this needed? The upstream project is archived

**Comment by JPeterMugaas** [comment]
from the readme file, "Patch ssl.match_hostname for Unicode(idna) domains support"

I think it adds support for something.

**Comment by lazka** [comment]
Yes, for older Python apparently.

To re-phrase: Why do _you_ need it?

**Comment by JPeterMugaas** [comment]
To be honest, I'm not sure.

**Pull request closed without merging**